packages/terra-dialog-modal/src/DialogModal.module.scss

```diff
@@ -71,15 +71,23 @@
     width: 100%; // Needed for IE to wrap content correctly in flexbox
   }
 
+  .dialog-modal-container:focus {
```

Comment:
Disabling the focus outline needs to be evaluated by UX/Accessibility to determine if focus is needed, and UX needs to determine if we want a stylized focus variant as well to match our themes.

Reply:
cc @neilpfeiffer

Reply:
@dkasper-was-taken - in terms of transferring programatic focus to the non-interactive parent content areas of our disclosure containers, adding `outline: none;` (no theme-ability needed) has been our standard practice like on [popup](https://github.com/cerner/terra-framework/blob/6cf7f8722f57836915c06589297b30efd09fb1df/packages/terra-popup/src/PopupContent.module.scss#L8), [slide-panel](https://github.com/cerner/terra-framework/blob/6cf7f8722f57836915c06589297b30efd09fb1df/packages/terra-slide-panel/src/SlidePanel.module.scss#L12), [abstract-modal](https://github.com/cerner/terra-framework/blob/6cf7f8722f57836915c06589297b30efd09fb1df/packages/terra-abstract-modal/src/AbstractModal.module.scss#L7), [the rollup-menu in tabs](https://github.com/cerner/terra-framework/blob/6cf7f8722f57836915c06589297b30efd09fb1df/packages/terra-tabs/src/StructuralTabs.module.scss#L186), etc.

This looks correct @saket2403